WordSearch.java

```diff
@@ -0,0 +1,44 @@
+// TIme Complexity: O(mxn(2^n))
```

Comment:
LC says: `Time Complexity: O(N⋅4 
L
 ) where NN is the number of cells in the board and LL is the length of the word to be matched.`